docs/FAQ.md

```diff
@@ -0,0 +1,106 @@
+
+## Do the GPU builds support quantized models?
+The default CUDA build does not support any quantized operators right now. The TensorRT EP has limited support for INT8 quantized ops. In general, support of quantized models through ORT is continuing to expand on a model-driven basis. For performance improvements, quantization is not always required, and we suggest trying alternative strategies to [performance tune](./ONNX_Runtime_Perf_Tuning.md) before determining that quantization is necessary.
+
+## Why is the GPU build slower than CPU?
+Depending on which execution provider you're using, it may not have full support for all the operators in your model. Fallback to CPU ops can cause hits in performance speed. Moreover even if an op is implemented by our CUDA execution provider, we may not necessarily assign/place the op to the CUDA EP due to performance reasons. To see the placement decided by ORT, turn on verbose logging and look at the console output.
+
+See:
+* [Issue 1368](https://github.com/microsoft/onnxruntime/issues/1368)
+* [Issue 1246](https://github.com/microsoft/onnxruntime/issues/1246)
+
+## My converted Tensorflow model is so slow - why?
+
+NCHW and NHWC are two different memory layout for 4-D tensors.
+
+Most TensorFlow operations used by a CNN support both NHWC and NCHW data format. The Tensorflow team suggests that on GPU NCHW is faster but on CPU NHWC is sometimes faster in Tensorflow. However, ONNX only supports NCHW. As a result, if the original model is in NHWC format, when the model is converted extra transposes may be added. The latest [keras-onnx converter](https://github.com/onnx/keras-onnx) can remove many of these transposes. If you can, consider retraining the model using NCHW.
+
+## Does the Python API add any overhead to performance?
+The Python API doesn't add any overhead to performance when using the CPU execution provider only. When using the GPU provider, inputs and outputs need to be copied from CPU to GPU and vice-versa and the Python API doesn't allow inputs/outputs to be setup for this to be done prior to execution (calling run). Hence this copy needs to be done on the fly as part of execution. This eats into the execution time thereby impacting performance.
+
+## Why is ORT slower than PyTorch when using GPU?
+Refer to above answer. Also see: 
+* [Issue 2404](https://github.com/microsoft/onnxruntime/issues/2404#issuecomment-554593653)
+* [Issue 2434](https://github.com/microsoft/onnxruntime/issues/2434)
+***
+
+# Other Questions and Issues
+
+## I can't build the [xyz] execution provider.
+Please read through the [BUILD.md](./../BUILD.md) page and makes sure your environment set up and dependency versions are all correct.
+
+## How do I turn on verbose mode logging in Python?
+```
+import onnxruntime as ort
+ort.set_default_logger_severity(0)
+```
+
+## How do I infer models that have multiple inputs and outputs using the C/C++ API?
+*[TODO: Add sample code]*
+
+See: 
```

Comment:
same as above. should summarize in the FAQ rather than force user to read a bunch of long issues

Reply:
We can remove this question until we add sample code in the repo.

Reply:
how long will it take to add sample code? if not too long, can we just have someone do it now otherwise it seems like something that'll just get put off. if it's more extensive, then ok.

Reply:
Actually I found an example that shows using multiple inputs/outputs. We can reference this. https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/test/shared_lib/test_inference.cc#L395